ingest/bulk_upload.py

```diff
@@ -75,16 +118,16 @@ def ICU_installed(es):
         for fn in args.filenames:
             files += glob.glob(fn)
 
-    debug("Will index {}".format(",".join(files)))
+    LOGGER.debug("Will index %s", ",".join(files))
 
     # Clear ES database if desired
     client = elasticsearch.client.IndicesClient(es)
     if clear_database:
         try:
-            debug("Will delete index " + INDEX_NAME)
+            LOGGER.debug("Will delete index %s", INDEX_NAME)
             client.delete(index=INDEX_NAME)
         except elasticsearch.exceptions.NotFoundError:
-            debug("Index not found, continuing")
+            LOGGER.debug("Index not found, continuing")
```

Comment:
Are all the uses of `LOGGER` for the new CRON job style we're doing or is it just for Docker?

Reply:
No, it's using the standard Python logging framework instead of print. This means we get niceness like log levels and timestamps. It's good whether we are using docker or not.